index.js

```diff
@@ -12,8 +12,44 @@ var unlink = RSVP.denodeify(fs.unlink);
 var tmpDir = require('os').tmpdir();
 var debug = require('debug')('async-disk-cache');
 var zlib = require('zlib');
+var heimdall = require('heimdalljs');
 
 var CacheEntry = require('./lib/cache-entry');
+var Metric = require('./lib/metric');
+
+heimdall.registerMonitor('async-disk-cache', function AsyncDiskCacheSchema() {});
```

Comment:
I did notice that before, but since the implementations are slightly different (in that we won't wrap methods more than once like FSMonitor can potentially do) that it would be unneeded. But I realize that if you happen to have this module in more than one location in your dep graph it can still happen.

Will update to fix, should be simple.